jobs/service-fabrik-broker/spec

```diff
@@ -142,6 +156,9 @@ properties:
   multi_az_enabled:
     description: "Switch used to turn on/off the multi-az support"
     default: false
+  ha_enabled:
```

Comment:
IMO, rather than having it as an explicit flag, it would be better to have it implicit. If the instance count is more than 1, we can implicitly assume that it is an HA setup. One less config and if someone makes mistake, won't be an issue.

Reply:
Completely agree with you. But keeping a different flag would give a notion of signalling directly rather than figuring out from #of instances. Looks cleaner for operator also.